etc/DependencyInstaller.sh

```diff
@@ -478,7 +485,17 @@ while [ "$#" -gt 0 ]; do
         -dev|-development)
             echo "The use of this flag is deprecated and will be removed soon"
             ;;
+        -base)
+            option="base"
+            ;;
+        -common)
+            option="common"
+            ;;
```

Comment:
Check if `option` is not `all` and give a warning about overwriting previous `-common` or `-base` argument.